packages/core/config/core-config-server-internal/src/index.ts

```diff
@@ -8,3 +8,4 @@
 
 export { coreDeprecationProvider } from './deprecation';
 export { ensureValidConfiguration } from './ensure_valid_configuration';
+export { getDeprecationsFor, getDeprecationsForGlobalSettings } from './test_utils';
```

Comment:
I don't like exporting those test utils from the `-internal` package, but we're using it both for the package's internal tests and for other tests, so we can't easily move it elsewhere... will have to do.